### ISSUE TITLE
Responsiveness

### ISSUE THREAD

**Issue by cvodehnal** [TheDanAlatriste/web-iv-portfolio-site#4]
Everything looks like it's going into place very nicely at each screen size. Great job! My only concern is that at mobile version on your project page, the thumbnails are getting a bit small, but I guess that up to you.

**Comment by cvodehnal** [comment]
I'd give you an advanced on this criteria.

**Comment by e-m-durham** [comment]
I agree with @cvodehnal about a few of the images on the project page. Also the columns on tablet on index.html begin to get really awkwardly sized, and incases text overlaps image, but maybe that will fix itself when you start working with the type.
